sdk/src/main/java/io/dapr/client/DaprClientHttpAdapter.java

```diff
@@ -192,21 +174,21 @@
       if (state.getEtag() != null && !state.getEtag().trim().isEmpty()) {
         headers.put(Constants.HEADER_HTTP_ETAG_ID, state.getEtag());
       }
-      String serializedKeyBody = objectSerializer.serialize(state.getKey());
+      String serializedKeyBody = state.getKey();
       serializedKeyBody += getOptionsAsQueryParameter(options);
       if (options.getConsistency() != null && !options.getConsistency().trim().isEmpty()) {
```

Comment:
This entire IF is no longer needed, all that logic is inside getOptionsAsQueryParameter method

Reply:
Ok. Will remove.